Jenkinsfile

```diff
@@ -129,7 +129,7 @@ def sonarAnalysis(args) {
 def hdbPodTemplate(closure) {
   podTemplate(
   containers: [
-    containerTemplate(name: 'hdb', image: 'wecharyu/hive-dev-box:executor', ttyEnabled: true, command: 'tini -- cat',
+    containerTemplate(name: 'hdb', image: 'ayushtkn/hive-dev-box:executor', ttyEnabled: true, command: 'tini -- cat',
```

Comment:
note: this definitely needs a follow-up (I won't block this change), because we still haven't taken care of adapting the hive-dev-box, so now we have different images on different branches, still no version-controlled Dockerfile + image in an official apache docker repo